.travis.yml

```diff
@@ -7,5 +7,9 @@ python:
   - "3.8"
   - "3.9-dev"
 
+install:
+  - pip install -r dev_requirements.txt
+
 script:
-- python setup.py test
+  - mypy meldebot/bot.py
```

Comment:
you can add --strict so it will fail instead of pass with warnings

Reply:
I will make it in another PR